packages/dd-trace/test/plugins/externals.json

```diff
@@ -86,5 +86,11 @@
       "name": "pg-native",
       "versions": ["3.0.0"]
     }
+  ],
+  "pino": [
```

Comment:
This is not needed since there is an actual integration for `pino-pretty` now, which will cause it to already be installed.

Reply:
I tried on a fresh repo to remove this and run the test with `yarn && PLUGINS=pino yarn test:plugins:ci` and it's failing the tests because it's missing `pino-pretty`. Either I'm missing a preliminary command, or it's the fact that `pino` itself is requiring the module (not our tests' code) that makes it not detect the automatically installed `pino-pretty` version. What do you think ?

Reply:
It should definitely work even without. There might be a different reason why the test is failing.